doc/src/compute_pressure_cartesian.rst

```diff
@@ -0,0 +1,63 @@
+.. index:: compute pressure/cartesian
```

Comment:
These files need to be fully integrated into the manual build by listing them in the `Commands_compute.rst` and the `compute.rst` pages. Also, src/.gitignore needs to be updated.